CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog and versioning
 
+## Unreleased
+
+### Breaking changes in this release
+
+- Synchronized consumer deprecated (#81)
```

Comment:
Should we also document deprecation of `create` api and mention `create_with_partitions`?